src/collection-view.js

```diff
@@ -295,7 +296,20 @@ Marionette.CollectionView = Marionette.AbstractView.extend({
       });
     }
 
-    return childView;
```

Comment:
I think @MeoMix ran into an isssue where there was a different Mn lib on the page whose views did not match